src/base/path.cpp

```diff
@@ -93,11 +114,17 @@ bool Path::isEmpty() const
 
 bool Path::isAbsolute() const
 {
+    // `QDir::isAbsolutePath` treats `:` as a path to QResource, so handle it manually
```

Comment:
Why?

Reply:
I don't know what you mean.

Reply:
**Why** do we need to handle it manually in a different way than it is currently handled?

Reply:
Do you expect `Path(":")` to be an absolute path?

Reply:
> Do you expect `Path(":")` to be an absolute path?

If we talk about `path to QResource` then I would expect it to be treated like Qt does.
But if such a path can represent a regular path as well, then we will have to make an explicit choice whether Path treats such paths as resource paths or as regular paths.

Reply:
IMO `Path` is considered a generic component and it shouldn't include the specifics of Qt like this. At least in this function I won't expect it to behave like `QDir`.
If resource paths are required to be checked then just use `QDir`.
Also this change shouldn't really affect the execution of qbt generally, because `Path` data can still be passed to other Qt functions (assuming there is no conditional related to this function call).

Reply:
> IMO `Path` is considered a generic component and it shouldn't include the specifics of Qt like this. At least in this function I won't expect it to behave like `QDir`.

Ok.


> assuming there is no conditional related to this function call

The code that expects the path to resources should ideally have an extra handling. So if it is not there, then it may have problems regardless of these changes.